.travis.yml

```diff
@@ -35,6 +35,11 @@ notifications:
       - "chat.freenode.net#oppia"
     on_success: never
     on_failure: always
+  webhooks:
+    urls:
+      - https://webhooks.gitter.im/e/f8f782497ec1ffcea2e7
```

Comment:
It can be found under Gitter integrations as: 

1. Click on the top-left settings button in gitter chat room
2. Click on Integrations
3. You'll see one integration with Travis integration and the unique webhook url mentioned.

Also worth noting that anyone can delete this, and it generates new unique url everytime.